### PR TITLE
Added FromJSON instance for 'EraInMode BabbageEra CardanoMode'

### PR DESCRIPTION
**Pull request by koslambrou**

### CODE REVIEW

File: cardano-api/src/Cardano/Api/Modes.hs [hunk 1]
```diff
@@ -205,6 +206,13 @@ instance FromJSON (EraInMode AlonzoEra CardanoMode) where
                          "parsing 'EraInMode AlonzoEra CardanoMode' failed, "
                          invalid
 
+instance FromJSON (EraInMode BabbageEra CardanoMode) where
```
[the remark this answers is not in the document]
Comment (koslambrou) [reply cardano-api/src/Cardano/Api/Modes.hs:209]: Sorry, I don't think I understand the question. I mean, a new era implies an additional pattern match statement here, no?

[the remark this answers is not in the document]
Comment (Jimbo4350) [reply cardano-api/src/Cardano/Api/Modes.hs:209]: Ah my bad. I didn't realize there were FromJSON instances defined for the other constructors.

**Review by Jimbo4350 — APPROVED** [review]
LGTM!

**Comment by Jimbo4350** [comment]
@koslambrou you should also update the `prop_json_roundtrip_eraInMode` test

**Comment by koslambrou** [comment]
@Jimbo4350 Thanks for that. I've updated the test.

**Comment by zmrocze** [comment]
Hey @Jimbo4350, I think it now only waits to be added to the bors queue? Or is it maybe merged somewhere else already?

This pr (and then [this update](https://github.com/input-output-hk/plutus-apps/blob/0eb87b0d52d5847867bfdb5fd34a9c8eb7c4a158/plutus-ledger/src/Ledger/Tx/CardanoAPI.hs#L240) in plutus-apps) would fix some of the tests in [plutip](https://github.com/mlabs-haskell/plutip).

**Comment by koslambrou** [comment]
Superseded by https://github.com/input-output-hk/cardano-node/pull/4237

**Pull request closed without merging**